DOCUMENTRETRIEVAL/documentRetrieval/library/postgresql.py

```diff
@@ -45,8 +42,11 @@ def db_return_docs_metadata(self, metric_fn_output):
             ids.append(tupl[0])
         if ids == []:
             raise Exception('No relavant documents for the given query.')
-        t = tuple(ids)
-        SQL = """SELECT document_id,document_source, date, title, celex_num, fulltextlink FROM documents WHERE document_id IN {}""".format(t)
+        if len(ids) == 1:
+            SQL = """SELECT document_id,document_source, date, title, celex_num, fulltextlink FROM documents WHERE document_id =""" + str(ids[0])
```

Comment:
Break the SQL statement into smaller strings. See previous comments on how to do that.